sdk/cosmosdb/cosmos/src/utils/batch.ts

```diff
@@ -36,10 +38,19 @@ export interface OperationBase {
   ifNoneMatch?: string;
 }
 
+export const BulkOperationType = {
+  Create: 'Create',
+  Upsert: 'Upsert',
+  Read: 'Read',
+  Delete: 'Delete',
+  Replace: 'Replace'
+}
```

Comment:
```suggestion
}  as const
```

Will get you the correct behavior here

Reply:
thanks, i was wondering why i couldnt use it in the types. Should fix all the weirdness I added

Reply:
hm, I still can't use it in types since it's a const not a type. I think I have to make the enum private to this file